spec/replication/primary_key_spec.rb

```diff
@@ -1,12 +1,18 @@
 describe "all tables" do
   let(:connection) { ApplicationRecord.connection }
 
-  it "have a primary key" do
+  it "have a primary key called id" do
     no_pk = []
     connection.tables.each do |t|
-      next if t == "schema_migrations"
-      no_pk << t unless connection.primary_key?(t)
+      next if %w(schema_migrations ar_internal_metadata).include?(t)
+      no_pk << t unless connection.primary_keys(t) == ["id"]
     end
-    expect(no_pk.size).to eq(0), "No primary key found for: #{no_pk.join(", ")}"
+    expect(no_pk.size).to eq(0), <<-EOS.gsub!(/^ +/, "")
+      Primary key "id" not found for the following table(s):
+
+      #{no_pk.join("\n")}
+
+      All tables must have a primary key called "id"
```

Comment:
You might want a more thorough description here...maybe something lke...

```
All tables must have a primary key called "id", as replication with pglogical requires that all tables being replicated have a primary key.

We have chosen to use "id" as the primary key, as opposed to a composite primary key, in order to avoid future migration problems should a table need to move away from a composite key in the future.
```

or something like that.